python-example-logs-api-extension/extensions/logs_api_http_extension/http_listener.py

```diff
@@ -2,26 +2,33 @@
 # SPDX-License-Identifier: MIT-0
 
 import json
+import os
 import sys
 from http.server import BaseHTTPRequestHandler, HTTPServer
 from threading import Event, Thread
 
 """ Demonstrates code to set up an HTTP listener and receive log events
 """
 
+RECEIVER_NAME = "sandbox"
 RECEIVER_IP = "0.0.0.0"
```

Comment:
I'd recommend renaming this as something that indicates that it's for local/test use. Maybe something like `DEBUG_RECEIVER_IP`?